identifier/backends/ldap/ldap.go

```diff
@@ -71,6 +72,9 @@ type LDAPIdentifierBackend struct {
 
 	timeout int
 	limiter *rate.Limiter
+
+	connLock sync.Mutex
```

Comment:
please call this field `mutex` similar like this is used with other structs in this project. It is unlikely that this struct will need multiple mutexes.